scipy/ndimage/filters.py

```diff
@@ -951,6 +952,8 @@ def maximum_filter1d(input, size, axis=-1, output=None,
 
 def _min_or_max_filter(input, size, footprint, structure, output, mode,
                        cval, origin, minimum):
+    if (size is not None) and (footprint is not None):
+        warnings.warn("Ignoring size because footprint is set", UserWarning, stacklevel=3)
```

Comment:
I think we don't normally capitalize warning or error messages, i.e. `Ignoring -> ignoring`.